packages/amplify-provider-awscloudformation/src/commands/awscloudformation/override.ts

```diff
@@ -3,17 +3,28 @@
 */
 
 import path from 'path';
-import { generateOverrideSkeleton, $TSContext } from 'amplify-cli-core';
+import { generateOverrideSkeleton, $TSContext, FeatureFlags } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+import * as fs from 'fs-extra';
 
 const subcommand = 'override';
 
 export const name = 'overrides';
 
 export const run = async (context: $TSContext) => {
-  const backendDir = context.amplify.pathManager.getBackendDirPath();
+  if (FeatureFlags.getBoolean('overrides.project')) {
+    const backendDir = context.amplify.pathManager.getBackendDirPath();
 
-  const destPath = path.normalize(path.join(backendDir, 'awscloudformation'));
-  const srcPath = path.normalize(path.join(__dirname, '..', '..', '..', 'resources', 'overrides-resource'));
-
-  await generateOverrideSkeleton(context, srcPath, destPath);
+    const destPath = path.normalize(path.join(backendDir, 'awscloudformation'));
+    const srcPath = path.normalize(path.join(__dirname, '..', '..', '..', 'resources', 'overrides-resource'));
+    // empty build dir for old CFN files
+    // no need for rollback since these files will be autogenerated after push
+    const rootStackBuildPath = path.join(destPath, 'build');
+    if (fs.existsSync(rootStackBuildPath)) {
+      fs.emptyDirSync(rootStackBuildPath);
```

Comment:
Why do we need to empty this? Can't we just override this? I'm a bit scared of having destroy operations of customer folders in code.

Reply:
I it fine to keep the old cfn file in their project when they migrate root stack?

Reply:
It would be safer to just remove the file and not the entire folder.